Sorstr.py

```diff
@@ -1,55 +0,0 @@
-from glob import glob
-
-import re
-
-
-class Sorstr:
-    inverted_index = {}
-
-    def index(self, pattern: str) -> None:
-        """
-        Index files indicated by the pattern.
-
-        :param pattern: The pattern to use to look for files 
-        """
-        self.inverted_index = {}
-
-        files = glob(pattern)
-        files.sort()
-        for file in files:
-            with open(file, 'r') as f:
-                contents = f.read().lower()
-                terms = re.split('[\s\W]+', contents)
-                terms = list(filter(None, terms))
```

Comment:
Interesting way of using `filter` by passing `None` to filter for truth. A more explicit filter function might improve legibility, though.